.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -674,6 +674,91 @@ stages:
         displayName: Publish NuGet packages
         artifact: nuget-packages
 
+
+- stage: build_dd_dotnet_windows
+  condition: and(succeeded(), eq(variables['isBenchmarksOnlyBuild'], 'False'))
+  dependsOn: [merge_commit_id]
+  variables:
+    targetShaId: $[ stageDependencies.merge_commit_id.fetch.outputs['set_sha.sha']]
+    targetBranch: $[ stageDependencies.merge_commit_id.fetch.outputs['set_sha.branch']]
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [build_dd_dotnet]
+
+  - job: build_dd_dotnet
+    timeoutInMinutes: 60 #default value
+
+    pool:
+      name: azure-windows-scale-set-3
+
+    steps:
+    - template: steps/clone-repo.yml
+      parameters:
+        targetShaId: $(targetShaId)
+        targetBranch: $(targetBranch)
+    - template: steps/install-latest-dotnet-sdk.yml
+  
+    - script: tracer\build.cmd BuildDdDotnet
+      displayName: Build dd-dotnet
+
+    - publish: $(artifacts)/dd-dotnet/win-x64
+      displayName: Uploading dd-dotnet win-x64
+      artifact: dd-dotnet-win-x64
```

Comment:
That's an interesting point. NativeAOT is not compatible with win-x86, and either way I don't think anybody is using a 32-bit version of Windows nowadays.
It's ok to use the x64 version for everything (it sets both `COR_PROFILER_PATH_32` and `COR_PROFILER_PATH_64`, so it will work regardless of the bitness of the target). However it can be slightly confusing for a customer not finding dd-dotnet.exe in the win-x86 folder of the tracer. We can probably add a readme file if anybody complains.